docs/notebooks/gait/gait_analysis.ipynb

```diff
@@ -9,7 +9,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 1,
```

Comment:
Mooi dat je hier een oplossing voor hebt gevonden ;)

Reply:
Yikes! Nieuwe commit heeft een _.gitattributes_ toegevoegd waar het nu wel gefixt moet zijn. Alle volgende commits van .ipynb files worden de outputs van verwijderd. Het kan zijn dat we dit uiteindelijk niet willen voor alle files, omdat bepaalde bestanden dataframes of plots gevisualiseerd hebben. Dit kunnen we vervolgens specificeren in de _.gitattributes_ dmv paths.

Reply:
Klinkt goed!